ci-operator/step-registry/ipi/conf/capability/baselinecaps/vset/ipi-conf-capability-baselinecaps-vset-commands.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+openshift-install version
+declare -a baseline_caps_set_array
+IFS="," read -r -a baseline_caps_set_array <<< "$(openshift-install explain installconfig.capabilities.baselineCapabilitySet | grep 'Valid Values' | awk -F':' '{print $2}' | tr -d '"')"
```

Comment:
I am thinking if there is a risk when `openshift-install explain` has bugs, missed some baseline caps?  Or shall we use a version to replace it?
```
baseline_caps_set_array=("vCurrent")

case ocp_major_version in
"11")
    baseline_caps_set_array=("vCurrent" "v4.11")
  ;;
"12")
    baseline_caps_set_array=("vCurrent" "v4.11" "v4.12")
  ;;
"13")
    baseline_caps_set_array=("vCurrent" "v4.11" "v4.12" "v4.13")
  ;;
"14")
    baseline_caps_set_array=("vCurrent" "v4.11" "v4.12" "v4.13" "v4.14")
  ;;
*)
  error out
  ;;
esac
```

Reply:
hmmm, seems that we don't detect issues on `openshift-installer explain` so far, but I'm okay to change the safe way you suggested.

Reply:
or:
```
v411_baseline_caps_set_array=("vCurrent" "v4.11")
v412_baseline_caps_set_array=(${v411_baseline_caps_set_array[@]} "v4.12")
v413_baseline_caps_set_array=(${v412_baseline_caps_set_array[@]} "v4.13")
v414_baseline_caps_set_array=(${v413_baseline_caps_set_array[@]} "v4.14")
v_current_version="v${ocp_major_version}${ocp_minor_version}_baseline_caps_set_array"
baseline_caps_set_array="${!v_current_version}"
```

Reply:
> seems that we don't detect issues on openshift-installer explain so far

Or we can keep the present codes, add one more pre-check for `openshift-install explain` output to ensure the OCP_VERSION info is shown up in the output.

Reply:
updated to use the second way to get expected set based on release image being tested.